akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -18,18 +18,18 @@ import scala.concurrent.duration.FiniteDuration
 import akka.japi.Util.immutableSeq
 
 object ClusterSettings {
-  type Team = String
+  type DataCenter = String
```

Comment:
Could we make it a value class (`extends AnyVal`) to get real type checking here?